### PR TITLE
Updates in the White Paper.

### PR DESCRIPTION
**Pull request by humb1t**
Signed-off-by: Nikita Puzankov <humb1t@yandex.ru>

### CODE REVIEW

File: docs/source/iroha_2_whitepaper.md [hunk 1]
```diff
@@ -1,137 +1,105 @@
...
++--+-+   ||Definition(s)|  |             |
+|Peer|   +--------------+  |             |
++--+-+   |                 |             |
+   |     +------------+    |             |
+   |     ||Account(s)||    | has   +-----v-----+
+   |     |------------------------->Signatories|
+   |     +-----------------+       +-----------+
+   |                       |             |
+   |                       |  has  +--------+
+   |                       +------->Asset(s)|
```
Comment (takemiyamakoto) [inline docs/source/iroha_2_whitepaper.md:73]: Shouldn't the concept of permissions be here somewhere?

Comment (humb1t) [reply docs/source/iroha_2_whitepaper.md:73]: In current impl we have permissions in form of assets, this solution is not accepted and we will discuss it in https://wiki.hyperledger.org/display/iroha/Permissions 

I decided to update current model in the whitepaper to be aligned with current state

**Comment by eadventurous** [comment]
@humb1t is it possible to remove the ae922be commit from this branch? I think it was added here by mistake.

**Comment by eadventurous** [comment]
When the commit is excluded I will approve as everything else seems to be good :smile:

**Comment by humb1t** [comment]
@eadventurous sry, but @takemiyamakoto should approve first - I will update PR after that

**Comment by eadventurous** [comment]
I also think that maybe this PR should go into `iroha2-wp` as I think the most recent updates to the whitepaper are there, am I right?

**Comment by takemiyamakoto** [comment]
I already merged the white paper parts, so can we close this?

**Comment by eadventurous** [comment]
Yes, let's close this PR then @humb1t .

**Pull request closed without merging**